### DISCUSSION TITLE
Is Emacs 29.x a hard requirement?

### DISCUSSION THREAD

**Discussion by jman-schief** [kickingvegas/casual-info#29] [Q&A]
Hi,

I've tried installing `casual-info` on Emacs 28.2 and I get the warning about requiring Emacs 29.1 (forgot the exact wording). I am installing `casual-info` from MELPA with `use-package`.

I was curious if this is a hard requirement. On some machines I use Debian stable (bookworm) and they still ship Emacs 28.2

thanks Charles for this useful package!

**Comment by kickingvegas** [comment]
@jman-schief 

Thanks for asking! TBH I’ve set 29.1 as the requirement because:

1.  Emacs 29.1 is the first release that included Transient as a built-in package.
2.  I started building Casual with 29.1 (now using 29.4).
3.  I don’t have the means to QA on earlier versions of Emacs.

All that said, something you can try is to install Magit 4.0 (either by package manager or manually) which will include the latest version of Transient. Then check out the actual repo of a Casual package and then include its `lisp` directory in your `load-path`. 

Please let me know if this works. I'd be interested in knowing.

**Reply by jman-schief** [reply]
In the end I decided to cheat and install Emacs 29.4 from the Debian backports to current stable (bookworm). Realistically I don't have time to dig into this :upside_down_face:

**Reply by kickingvegas** [reply]
In many ways outside of Casual, I think you'll be happier that you've upgraded.
